README.md

```diff
@@ -11,7 +11,34 @@ Project goals:
 
 This software should be able to run for weeks on end without problems.
 
-## Setup
+## Development Setup
+Prerequisites:
+* Docker
+* Node.js
+
+1. Copy `(project-root)/conf/docker-config.json` to `(project-root)/config.json`.
+2. Start the MongoDB instance by running `npm mongo-docker`.
+3. The monitor and web interface by running `npm node-docker-win` (Windows) or `npm node-docker-nix` (Linux and OS X).
+4. Access the web interface at `http://localhost:8081`.
```

Comment:
I'm not sure if the docker image would be accessible at this step if the user followed these instructions.